examples/benchmarks/main.py

```diff
@@ -3,10 +3,12 @@
 import matplotlib
 import matplotlib.pyplot as plt
 import numpy as np
-from prio import prio
+from prio_processor.prio import wrapper as prio
+from prio import PrioContext
 from tqdm import tqdm
 
 
+@PrioContext()
```

Comment:
Likewise, it seems ashame we need to use a decorator here but I trust your judgement if you say it's for the best.